bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/impl/BahmniDrugOrderServiceImpl.java

```diff
@@ -167,6 +178,110 @@ public List<Order> getAllDrugOrders(String patientUuid, String patientProgramUui
+        String prescriptionString = "";
+        int counter = 1;
+        for (Map.Entry<BahmniDrugOrder, Integer> entry : drugOrderDurationMap.entrySet()) {
+            prescriptionString += counter++ + ". " + getDrugOrderString(entry.getKey(), drugOrderDurationMap.get(entry.getKey()), locale) + "\n";
+        }
+        return prescriptionString;
+    }
+
+    @Override
+    public String getAllProviderAsString(List<BahmniDrugOrder> drugOrders) {
+        Set providerSet = new LinkedHashSet();
+        for (BahmniDrugOrder drugOrder : drugOrders) {
+            providerSet.add("Dr " + drugOrder.getProvider().getName());
+        }
+        return StringUtils.collectionToCommaDelimitedString(providerSet);
+    }
+
+    private String getDrugOrderString(BahmniDrugOrder drugOrder, Integer duration, Locale locale) {
+        String drugOrderString = drugOrder.getDrug().getName();
+        drugOrderString += ", " + (drugOrder.getDosingInstructions().getDose().intValue()) + " " + drugOrder.getDosingInstructions().getDoseUnits();
+        drugOrderString += ", " + drugOrder.getDosingInstructions().getFrequency() + "-" + duration.toString() + " " + drugOrder.getDurationUnits();
+        drugOrderString += ", start from " + convertUTCToGivenFormat(drugOrder.getEffectiveStartDate(), "dd-MM-yyyy", Context.getMessageSourceService().getMessage(SMS_TIMEZONE, null, locale));
+        if(drugOrder.getDateStopped() != null)
+            drugOrderString += ", stopped on " + convertUTCToGivenFormat(drugOrder.getDateStopped(), "dd-MM-yyyy", Context.getMessageSourceService().getMessage(SMS_TIMEZONE, null, locale));
+        return drugOrderString;
+    }
+
+    private Boolean areValuesEqual(Object value1, Object value2) {
+        if(value1 != null && value2 != null) {
+            return value1.equals(value2);
+        }
+        return (value1 == null && value2 == null);
+    };
+
+    private Boolean compareDosingInstruction(EncounterTransaction.DosingInstructions value1, EncounterTransaction.DosingInstructions value2) {
+        String doseAndFrequency1 = value1.getDose() + " " + value1.getDoseUnits() + ", " + value1.getFrequency();
+        String doseAndFrequency2 = value2.getDose() + " " + value2.getDoseUnits() + ", " + value2.getFrequency();
+        return doseAndFrequency1.equals(doseAndFrequency2);
+    };
+
+    private double getDateDifferenceInDays(Date date1, Date date2){
+        long diff = date2.getTime() - date1.getTime();
+        return (diff / (1000*60*60*24));
+    }
+
+    private Collection<Concept> getOrdAttributeConcepts() {
+        Concept orderAttribute = conceptService.getConceptByName(BahmniOrderAttribute.ORDER_ATTRIBUTES_CONCEPT_SET_NAME);
+        return orderAttribute == null ? Collections.EMPTY_LIST : orderAttribute.getSetMembers();
+    }
+
+    private List<BahmniDrugOrder> getBahmniDrugOrdersForVisit(String patientUuid, List<DrugOrder> drugOrders) {
```

Comment:
rename getDrugOrdersForVisit()

Reply:
removed this method as it no longer needed